platform-ui-skin/src/main/webapp/skin/less/core/helpers.less

```
@@ -97,6 +97,9 @@
 .text-font-size {
   font-size: @fontSizeLarge !important;
 }
+.text-header {
+  font-size: @sizeIconDefault !important;
```

Please add a significant dedicated variable instead of reusing the same variable for another type of element

```suggestion
  font-size: @textHeaderFontSize !important;
```

---

done!